webrtc.html

```diff
@@ -4967,11 +4967,11 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       </dl>
 
       <dl class="idl" title="dictionary RTCDataChannelInit">
-        <dt>boolean ordered = true</dt>
+        <dt>(boolean or RTCDataChannelOrdered) ordered = "ordered"</dt>
 
         <dd>
-          <p>If set to false, data is allowed to be delivered out of order. The
-          default value of true, guarantees that data will be delivered in
+          <p>If set to <code>"unordered"</code> (or <code>false</code> for backwards compatibility), data is allowed to be delivered out of order. The
+          default value of <code>"ordered"</code> (or <code>true</code> for backwards compatibility), guarantees that data will be delivered in
           order.</p>
```

Comment:
I thought we weren't putting backwards compatibility in the spec, and just letting the browsers handle that.  However, I think it might be good to call this "for convenience", since it's easier to remember to say "ordered: false" than "ordered: unordered"